doc/modules/decomposition.rst

```diff
@@ -864,16 +864,47 @@ The graphical model of LDA is a three-level Bayesian model:
 .. image:: ../images/lda_model_graph.png
    :align: center
 
+
+Note on notations presented in the graphical model above:
+
+  * The corpus is a collection of :math:`D` documents.
+  * A document is a sequence of :math:`N` words.
+  * There are :math:`K` topics in the corpus. 
+  * The boxes represent repeated sampling. 
+
+
+The three levels of the Bayesian model are: 
```

Comment:
I don't think you should call it bayesian, except if "Bayesian network". It is a bayes net, but does not really correspond to Bayesian statistics in the sense that we do not care about the full posterior, nor do we have educated priors.

Reply:
I'm not sure these levels are needed beyond the generative process description below...?

Reply:
removed description on the 3 levels